yt/yt/core/logging/unittests/logging_ut.cpp

```diff
@@ -657,13 +666,14 @@ TEST_F(TLoggingTest, StructuredLoggingJsonFormat)
     auto formatter = std::make_unique<TStructuredLogFormatter>(
         ELogFormat::Json,
         /*commonFields*/ THashMap<TString, INodePtr>{},
-        /*enableControlMessages*/ true,
         /*enableSourceLocation*/ false,
         /*enableSystemFields*/ true,
+        /*enableHostField*/ false,
         jsonFormat);
 
     auto writer = CreateFileLogWriter(
         std::move(formatter),
+        CreateDefaultSystemLogEventProvider(writerConfig),
```

Comment:
Please also cherry-pick this fix: https://github.com/ytsaurus/ytsaurus/commit/13d87d4aa9b78d1ac94d0e0d00633b4d40b5ae05

Reply:
And done.